pkgs/leak_tracker/lib/src/leak_tracking/_primitives/model.dart

```diff
@@ -109,17 +109,17 @@ class LeakSkipList {
 }
 
 /// Skip lists for each type of leak.
```

Comment:
nit: update dart-doc. "The total set of skipped leaks for both [nonGCed] and [notDisposed] leaks."